include/revng/PipelineC/Tracing/Runner.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/Error.h"
+
+#include "revng/PipelineC/Tracing/TraceFile.h"
+
+namespace revng::tracing {
+llvm::Error runTrace(const revng::tracing::Trace &Trace);
```

Comment:
This should either become a private function of a `Main.cpp` or a method of `Trace`. Same for everything in `Inspector.h`.